packages/asset-assembler/src/index.ts

```diff
@@ -0,0 +1,92 @@
+'use strict';
```

Comment:
similarly should we have comments for each export: some of them are not obvious from the variable name.

Reply:
Done! Added comments to all exported types/functions.